Workiva/over_react_codemod
# FED-1979 Add suggestor to replace null argument in dom callback

## Motivation
  

## Changes
  

#### Release Notes
  

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
        - [ ] Run the following commands on any of the repos in [this SG query](https://sourcegraph.plat.workiva.net/search?q=context:global+lang:dart+%5C.%28onAnimationEnd%7ConAnimationIteration%7ConAnimationStart%7ConCopy%7ConCut%7ConPaste%7ConKeyDown%7ConKeyPress%7ConKeyUp%7ConFocus%7ConBlur%7ConChange%7ConInput%7ConSubmit%7ConReset%7ConClick%7ConContextMenu%7ConDoubleClick%7ConDrag%7ConDragEnd%7ConDragEnter%7ConDragExit%7ConDragLeave%7ConDragOver%7ConDragStart%7ConDrop%7ConMouseDown%7ConMouseEnter%7ConMouseLeave%7ConMouseMove%7ConMouseOut%7ConMouseOver%7ConMouseUp%7ConPointerCancel%7ConPointerDown%7ConPointerEnter%7ConPointerLeave%7ConPointerMove%7ConPointerOver%7ConPointerOut%7ConPointerUp%7ConTouchCancel%7ConTouchEnd%7ConTouchMove%7ConTouchStart%7ConTransitionEnd%7ConScroll%7ConWheel%7ConCopyCapture%7ConCutCapture%7ConPasteCapture%7ConKeyDownCapture%7ConKeyPressCapture%7ConKeyUpCapture%7ConFocusCapture%7ConBlurCapture%7ConChangeCapture%7ConInputCapture%7ConSubmitCapture%7ConResetCapture%7ConClickCapture%7ConContextMenuCapture%7ConDoubleClickCapture%7ConDragCapture%7ConDragEndCapture%7ConDragEnterCapture%7ConDragExitCapture%7ConDragLeaveCapture%7ConDragOverCapture%7ConDragStartCapture%7ConDropCapture%7ConMouseDownCapture%7ConMouseEnterCapture%7ConMouseLeaveCapture%7ConMouseMoveCapture%7ConMouseOutCapture%7ConMouseOverCapture%7ConMouseUpCapture%7ConGotPointerCapture%7ConLostPointerCapture%7ConTouchCancelCapture%7ConTouchEndCapture%7ConTouchMoveCapture%7ConTouchStartCapture%7ConScrollCapture%7ConWheelCapture%29%5C%28null%5C%29+-repo:over_react&patternType=regexp&sm=1&groupBy=repo) locally
        ```
        dart pub global activate https://github.com/Workiva/over_react_codemod.git --source=git --git-ref=batch/fedx/FED-1979_dom_callback_null_args
        dart pub global run over_react_codemod:dom_callback_null_args --yes-to-all
        ```
        and verify that the dom callbacks that used to pass `null` literal as an argument, now have the correct empty synthetic event being passed instead.

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:


[contributing-review-types]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#manual-testing-criteria
